prime-dotnet-webapi/Configuration/Database/Seeded Tables/CollegeConfiguration.cs

```diff
@@ -10,27 +10,27 @@ public override IEnumerable<College> SeedData
-                    new College { Code = 17, Name = "College of Speech and Hearing Health Professionals of BC"                       },
-                    new College { Code = 18, Name = "College of Traditional Chinese Medicine Practitioners and Acupuncturists of BC" },
-
-                    new College { Code = 19, Name = "BC College of Social Workers" }
+                    new College { Code = 4,  Name = "College of Chiropractors of BC", Weight = 40                                                  },
+                    new College { Code = 5,  Name = "College of Dental Hygenists of BC", Weight = 999                                              },
+                    new College { Code = 6,  Name = "College of Dental Technicians of BC", Weight = 999                                            },
+                    new College { Code = 7,  Name = "College of Dental Surgeons of BC", Weight = 999                                               },
+                    new College { Code = 8,  Name = "College of Denturists of BC", Weight = 60                                                     },
+                    new College { Code = 9,  Name = "College of Dietitians of BC", Weight = 70                                                     },
+                    new College { Code = 10, Name = "College of Massage Therapists of BC", Weight = 80                                             },
+                    new College { Code = 11, Name = "College of Naturopathic Physicians of BC", Weight = 90                                        },
+                    new College { Code = 12, Name = "College of Occupational Therapists of BC", Weight = 100                                       },
+                    new College { Code = 13, Name = "College of Opticians of BC", Weight = 110                                                     },
+                    new College { Code = 14, Name = "College of Optometrists of BC", Weight = 120                                                  },
+                    new College { Code = 15, Name = "College of Physical Therapists of BC", Weight = 130                                           },
+                    new College { Code = 16, Name = "College of Psychologists of BC", Weight = 140                                                 },
+                    new College { Code = 17, Name = "College of Speech and Hearing Health Professionals of BC", Weight = 160                       },
+                    new College { Code = 18, Name = "College of Traditional Chinese Medicine Practitioners and Acupuncturists of BC", Weight = 170 },
+                    new College { Code = 19, Name = "BC College of Social Workers", Weight = 180                                                   },
+                    new College { Code = 20, Name = "College of Oral Health Professionals", Weight = 50                                            }
```

Comment:
Considering that there are 4 enrollments from College of Chiropractors vs. 125 Production licences from the four previous dental colleges, College of Oral Health Professionals should be listed higher
![image](https://github.com/bcgov/moh-prime/assets/7586665/abd0df8e-3174-482f-867d-e6ef95782885)